### ISSUE TITLE
Update schmittjoh/serializer to v3

### ISSUE THREAD

**Issue by davidspiess** [HGV/mss-php#3]
As already discussed in #2 we should eventually upgrade to v3.
@TomOne do you remember what problem you encountered when you tried to upgrade?

**Comment by TomOne** [comment]
With jms/serializer v3, `@Inline` no longer works in combination with `@XmlList`, e. g.:

https://github.com/HGV/mss-php/blob/9e4c7a9593ce9e9178592763080c8161bcbc9c4c/src/Schema/Response/FeaturesView.php#L9-L17

With v1, the output is:

```JSON
{ "features_view": [{ "id": 1, "title": "Lorem ipsum" }] }
```

With v3, the output is:

```JSON
{ "features_view": { "feature": [] } }
```

Possibly related issue: https://github.com/schmittjoh/serializer/issues/1088

**Comment by TomOne** [comment]
Solved with v3.0.0.
